segmenters/nlp/Sentencizer/__init__.py

```diff
@@ -12,7 +12,8 @@ class Sentencizer(BaseSegmenter):
     The text is split by the punctuation characters listed in ``punct_chars``.
     The sentences that are shorter than the ``min_sent_len``
     or longer than the ``max_sent_len`` after stripping will be discarded.
-
+    
+    :param lang: language of the input text, by default "en".
```

Comment:
I agree it is better to remove the check than to add this language parameter

Reply:
@JoanFM New commit that removes the filter. 
CI/hub-build-test is failing, any pointers why? I am a new contributor so not sure why it might be happening :/

Reply:
hmmm, it was complaining nothing to build. No worry. You need to bump the version number from `0.0.10` to `0.0.11` to trigger the building process.

https://github.com/haroonrashid235/jina-hub/blob/fix-sentencizer-segment-text-5141/segmenters/nlp/Sentencizer/manifest.yml#L11